javascript/app.js

```diff
@@ -0,0 +1,254 @@
+// 		return `1 is a special case!`;
+// 	}
+// 	for (let i = 2; i < num; i++) {
+// 		if (num % i === 0) {
+// 			return false;
+// 		}
+// 		return num !== 1;
+// 	}
+// }
+// console.log(checkPrime(13));
+
+// Step Two
+// Write another function called printPrimes that will print (console log) all the Primes up to an arbitrary limit. For example, if you invoke your function with printPrimes(97), it will print all the Prime numbers up to and including 97. This function can call on the previous checkPrime function.
+
+const checkPrime = (num) => {
+	// Start with 2 because 1 is its own special special case
+	if (num === 1) {
+		return `1 is a special case!`;
+	}
+	for (let i = 2; i < num; i++) {
+		if (num % i === 0) {
+			return false;
+		}
+	}
+	return num !== 1;
+
+}
+
+// Create array that will hold values that will be found when checkPrime is invoked in printPrime
+const listPrimes = [];
+
+const printPrimes = (numLimit) => {
+	// loop up to the number and check each one
+	for (let j = 1; j <= numLimit; j++) {
+		// CALL FUNCTION to make it available!
+		checkPrime(j);
+		if (checkPrime(j) === true) {
+			listPrimes.push(j);
+		}
+	}
+	console.log(listPrimes);
+}
+// console.log(checkPrime(13));
+console.log(printPrimes(97));
+
+// 🔴 **Commit your work.** 
+// The commit message should read: 
+// "Commit 6 - Prime Numbers".
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+
+
+// CSS
+// Watch the following three videos on CSS:
+
+// First CSS video - 5 minutes
+// Second CSS video - 11 minutes
+// Third CSS video - 17 minutes
+// You will need to have watched these videos for tomorrow's lab.
+
+// Hungry for more?
+// Complete the afternoon lab.
+
+// Write a function insertDash that accepts a number as a parameter and returns a string with a dash inserted between any consecutive odd numbers. There should not be a dash at the end, it goes only between numbers.
+
+// console.log(insertDash(454793));
+
+// => 4547-9-3
+// Commit.
+// Write a function reverseString that takes a string as a parameter and returns that string with the letters reversed without using .split(), .reverse(), or .join().
+
+// Commit.
+
+// Make your palindrome function from problem two above work regardless of spacing (or capitalization). So, for example, "Sit on a potato pan Otis" or "Bird rib" would pass the test.
+
+// Commit.
+
+// Make your palindrome function work even if the string contains punctuation. So: "Sit on a potato pan, Otis!!!" or "A man, a plan, a canal: Panama." would pass the test.
+
+// Commit.
+
+// Make a "word palindrome" function that returns true if the words in a phrase are the same backwards and forwards. It should not care about spacing, capitalization, or punctuation. For example the following string would pass the test:
+
+// "Son, I am able," she said. "Though you scare me, watch!" said I, "Beloved," I said, "watch me scare you!" Though, said she: "able am I, son."
+
+// Commit.
```

Comment:
if you get done w/ hw4 this weekend, try these palindrome ones.  i think you'd enjoy